doc/source/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.md

```diff
@@ -278,50 +255,3 @@ NAME                                                      READY   STATUS    REST
 prod-pytorch-text-classifier-gkp9b-r9k5r                  1/1     Running   0          5s
 torch-text-classifier-gkp9b-raycluster-s2f65-head-hfvht   1/1     Running   0          35s
 ```
-
```

Comment:
Step 1 ~ Step 6 are enough to demonstrate priority scheduling, so I remove this section to simplify the tutorial.